### ISSUE TITLE
"error 400: unparsable"

### ISSUE THREAD

**Issue by alaz** [softprops/ls#45]
Hello,

When trying to "lsync" this [project](https://github.com/osinka/subset), I get

``` bash
> lsync
[info] lsyncing project osinka/subset@0.6.3...
[warn] Error synchronizing project libraries Unexpected response code: 400
[warn] Unparsable
[success] Total time: 2 s, completed 22.01.2012 0:23:43
```

Can you please take a look?

Regards,
Alexander

**Comment by softprops** [comment]
Sorry about that. This is a bug. I'm currently doing some caveman style json serialization which isn't encoding multi-line strings properly in json. I'll have a look now and see if I can't push the fix this weekend in a new release. I'll ping you when I do.

**Comment by alaz** [comment]
Thanks!

**Comment by alaz** [comment]
One line limit encourages brevity -- I've managed to express the project description on one line. No need to hurry with the fix!

**Comment by softprops** [comment]
Thanks @alaz the fix for this is actually in master but I have seeing another issue class loader issue conflict between jerkson json serialization and sbt's weird classloader segregation at runtime which worried me too much to do a quick release. I want to get to the bottom of that before I do the next release but this json serialization fix will definitely be in the next release. Again, thats for reporting it and sorry for causing you grief.

**Comment by alaz** [comment]
No problem at all! I'm really happy with the shorter project description, so this is for the better

**Comment by softprops** [comment]
should be fixed in 0.12 branch
